frontend/src/lib/components/icons.tsx

```diff
@@ -1551,10 +1551,10 @@ export function IconLink(props: React.SVGProps<SVGSVGElement>): JSX.Element {
 
 export function IconMessages(props: React.SVGProps<SVGSVGElement>): JSX.Element {
     return (
-        <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
I think this makes it worse actually, because now the icon contents are off-center:
<img width="79" alt="Screen Shot 2022-08-01 at 17 26 52" src="https://user-images.githubusercontent.com/4550621/182184608-aa1ef8e2-fc5f-4e21-8ca6-0540721687c0.png">
Ideally this JSX would be just this SVG: https://fonts.gstatic.com/s/i/materialiconsoutlined/question_answer/v12/24px.svg, with `fill="currentColor"`

Reply:
Ok let me fix that, I think the icons in general need to be worked on, this is not the only icon that has this kind of problems :/